src/seedu/addressbook/data/person/Address.java

```diff
@@ -36,8 +36,8 @@ public static boolean isValidAddress(String test) {
     }
 
     @Override
```

Comment:
Good job on retaining `@Override`, since it denotes an instance method that overrides a method declared in an interface.